aws/biganimal_AWS_basepolicy_v2.json

```diff
@@ -0,0 +1,297 @@
+                }
+            },
+            "Sid": "BigAnimalEC2UpdatePolicy"
+        },
+        {
+            "Action": [
+                "ecs:DescribeTasks",
+                "ecs:ListTasks"
+            ],
+            "Effect": "Allow",
+            "Resource": "*",
+            "Condition": {
+                "ArnEquals": {
+                    "ecs:cluster": "arn:aws:ecs:*:*:cluster/BigAnimal*"
+                }
+            },
+            "Sid": "BigAnimalECSReadTaskPolicy"
+        },
+        {
+            "Action": [
+                "ecs:TagResource"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:ecs:*:*:cluster/BigAnimal*",
+            "Condition": {
+                "StringEquals": {
+                    "aws:RequestTag/ManagedBy": "BigAnimal"
+                }
+            },
+            "Sid": "BigAnimalECSTagPolicy"
+        },
+        {
+            "Action": [
+                "ecs:DeleteCluster",
+                "ecs:DescribeClusters"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:ecs:*:*:cluster/BigAnimal*",
+            "Sid": "BigAnimalECSClustersPolicy"
+        },
+        {
+            "Action": [
+                "ecs:RunTask"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+                "arn:aws:ecs:*:*:task-definition/dp*",
+                "arn:aws:ecs:*:*:task-definition/rr*"
+            ],
+            "Condition": {
+                "ArnEquals": {
+                    "ecs:cluster": "arn:aws:ecs:*:*:cluster/BigAnimal*"
+                }
+            },
+            "Sid": "BigAnimalECSRunTaskPolicy"
+        },
+        {
+            "Action": [
+                "iam:CreateOpenIDConnectProvider",
+                "iam:DeleteOpenIDConnectProvider",
+                "iam:GetOpenIDConnectProvider",
+                "iam:TagOpenIDConnectProvider",
+                "iam:UntagOpenIDConnectProvider"
+            ],
+            "Effect": "Allow",
+            "Resource": "arn:aws:iam::*:oidc-provider/oidc.eks.*.amazonaws.com*",
+            "Condition": {
+                "StringEquals": {
+                    "aws:ResourceTag/ManagedBy": "BigAnimal"
+                }
+            },
+            "Sid": "BigAnimalOpenIDConnectProviderPolicy"
+        },
+        {
+            "Action": [
+                "kms:CreateGrant",
+                "kms:DeleteAlias",
+                "kms:DescribeKey",
+                "kms:EnableKeyRotation",
+                "kms:GetKeyPolicy",
+                "kms:GetKeyRotationStatus",
+                "kms:ScheduleKeyDeletion",
+                "kms:TagResource",
+                "kms:UpdateAlias"
+            ],
+            "Effect": "Allow",
+            "Resource": [
+                "arn:aws:kms:*:*:key/*",
+                "arn:aws:kms:*:*:alias/kms-cmk-*"
+            ],
+            "Sid": "BigAnimalKMSPolicy"
+        },
```

Comment:
Have we tried to add a `Condition` here like
`
            "Condition": {
                "StringEquals": {
                    "aws:ResourceTag/ManagedBy": "BigAnimal"
                }
            },
`?

Reply:
@Simonzhaohui do we have the tag `ManagedBy:BigAnimal` for KMS resources?